pkg/issuer/acme/dns/route53/route53.go

```diff
@@ -98,9 +97,14 @@ func (d *sessionProvider) GetSession(ctx context.Context) (aws.Config, error) {
 		return aws.Config{}, fmt.Errorf("unable to create aws config: %s", err)
 	}
 
+	// Explicitly set the region to aws-global so that AssumeRole can be used
+	// with the global sts endpoint.
```

Comment:
Write more here to summarize our understanding of the situation. 
I.e. that we think the original behaviour in cert-manager 1.14 (which used aws-sdk-v1) was to always use the global STS endpoint, regardless of the region supplied by the user or by ambient configuration.
Explain that we set the sts client to always use `aws-global` to maintain backwards compatibility.

And maybe add a  TODO to remind us that some users may prefer to connect to a regional STS endpoint,
and that we are not sure if the user will necessarily always want to connect to STS in the same region as  their route53 service.

Reply:
I updated to comment to include the reason (backwards compatibility with 1.14).